packages/cli/templates/cloudflare/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "tmp-remix-app",
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "scripts": {
+    "build": "remix vite:build",
+    "deploy": "npm run build && wrangler pages deploy ./build/client",
+    "dev": "remix vite:dev",
+    "start": "wrangler pages dev ./build/client",
+    "typegen": "wrangler types",
+    "preview": "npm run build && wrangler pages dev ./build/client",
+    "build-cf-types": "wrangler types"
+  },
+  "dependencies": {
+    "@remix-run/cloudflare": "^2.8.1",
+    "@remix-run/cloudflare-pages": "^2.8.1",
+    "isbot": "^4.1.0",
+    "miniflare": "^3.20231030.4"
+  },
+  "devDependencies": {
+    "@cloudflare/workers-types": "^4.20240405.0",
+    "vite-tsconfig-paths": "^4.2.1",
+    "wrangler": "^3.48.0"
```

Comment:
crap we have 2 versions of wrangler now and remix has peer dependency on it
try to add it also to root deps, maybe will help to avoid duplicating remix

Reply:
Fixed